functional_tests/scylla_operator/test_functional.py

```diff
@@ -206,13 +206,18 @@ def get_pod_names_and_ips(cluster_name: str, namespace: str):
                         assert row["rack"] == k8s_cluster.rack_name
                         assert row["data_center"] == ip_to_dc_map[row["peer"]]
             need_to_collect_logs = False
+            log.info("DB cluster's peers info is correct")
+            return None
         finally:
             if need_to_collect_logs:
                 KubernetesOps.gather_k8s_logs(
                     logdir_path=logdir, kubectl=kubectl, namespaces=[namespace, namespace2])
                 need_to_collect_logs = False
             k8s_cluster.helm(f"uninstall {target_chart_name2} --timeout 120s", namespace=namespace2)
-            kubectl(f"delete namespace {namespace2}", ignore_status=True, timeout=120)
+            try:
+                kubectl(f"delete namespace {namespace2}", ignore_status=True, timeout=120)
```

Comment:
i think a higher timeout would fit better - finalizers and gc can take more time then 2 minutes (takes time on my local VM as well)

or at least use `--wait=false` but check that status that the delete call has succeeded